packages/core/src/interfaces/components.ts

```diff
@@ -109,6 +109,10 @@ export interface ZoomBarsOptions {
 	 * currently only the top position is supported
 	 */
 	top?: ZoomBarOptions;
+	/**
+	 * whether keep updating range axis in real time while zoom domain is changing
+	 */
+	updateRangeAxis?: boolean;
```

Comment:
I've already commented on this before, not sure where my comment went (usually they disappear if you force-push over the branch and the lines that were commented on get removed).

This seems like it should just be flag under:
```js
{
  axes: {
      "top|left|bottom|right": {
         update: boolean;
      }
  }
}
```

whether it needs to be called `update` or `updateWhenZooming` depends on whether this only applies to zooming or not

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/766/commits/a224ab3561811cf0d6454093d2fb3af0fe5ab303.